### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by BenIraa**

### CODE REVIEW

**Review by Gedeonir — COMMENTED** [review]
reviewed

File: package.json [hunk 1]
```diff
@@ -34,7 +36,6 @@
     "react-test-renderer": "^18.0.0",
     "regenerator-runtime": "^0.13.9"
   },
-
```
Comment (kareraolivier) [inline package.json:37]: remove this space bro

**Review by kareraolivier — CHANGES_REQUESTED** [review]

**Review by kareraolivier — APPROVED** [review]

File: .github/workflows/Ci.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ CI ]
```
Comment (mansurissa) [inline .github/workflows/Ci.yml]: replace CI with develop for the branch to be checked

**Review by mansurissa — CHANGES_REQUESTED** [review]

File: .github/workflows/Ci.yml (hunk 2 above)
Comment (BenIraa) [reply .github/workflows/Ci.yml]: Oops Let me change

**Review by mansurissa — APPROVED** [review]
Great

**Pull request merged**